src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -495,7 +495,121 @@ end
 
 
 ############################
-### 5: Display
+# 5: Toric varieties from triangulations
+############################
+
+@doc Markdown.doc"""
+    NormalToricVarietyFromTriangulation(P::Polyhedron)
+
+Returns the list of toric varieties obtained from fine regular
+star triangulations of the polyhedron P.
+
+# Examples
+```jldoctest
+julia> P = convex_hull([0 0 0; 0 0 1; 1 0 1; 1 1 1; 0 1 1])
+A polyhedron in ambient dimension 3
+
+julia> NormalToricVarietyFromTriangulation(P::Polyhedron)
+2-element Vector{NormalToricVariety}:
+ A normal toric variety
+ A normal toric variety
+```
+"""
+function NormalToricVarietyFromTriangulation(P::Polyhedron)
```

Comment:
I would suggest a different name, even though it becomes bulky that way:
```suggestion
function NormalToricVarietiesFromStarTriangulations(P::Polyhedron)
```
1. It probably returns more than one variety most of the time.
2. It computes many triangulations.

Maybe one could introduce a helper function `NormalToricVarietyFromStarTriangulation` that takes two arguments, the points + maximal cones. Only if useful.

Reply:
Agreed.

Even more, it would be good to have a method that accept a number k and then returns k such varieties. (Or precisely: AT most k. In principle, one could have k larger than the number of triangulations, in which case one would return all triangulations.) Again, I suggest this for another PR.